sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -779,6 +789,19 @@ Exceptions:
   the same order.
 |===
 
+==== Graph Properties
+
+There is the one following property defined that can be passed to a `command_graph`
+on construction via the property list parameter.
+
+1. `property::graph::no_cycle_check` - This property disables any checks if
```

Comment:
I think we'll need a new error listed in the Exceptions description of `make_edge()` to say that it will throw an error if adding an edge creates a cycle. This is the error checking behaviour requested, that using this property will disable.